src/SourceLocator/Type/AggregateSourceLocator.php

```diff
@@ -24,8 +24,10 @@ public function __construct(array $sourceLocators = [])
     {
         // This slightly confusing code simply type-checks the $sourceLocators
         // array by unpacking them and splatting them in the closure.
-        $validator            = static fn (SourceLocator ...$sourceLocator): array => $sourceLocator;
-        $this->sourceLocators = $validator(...$sourceLocators);
+        $validator = static fn (SourceLocator ...$sourceLocator): array => $sourceLocator;
```

Comment:
We can probably drop this validation

Reply:
https://github.com/Roave/BetterReflection/pull/1012/commits/d512fe26b7559d973a6b4d5ee739d7c42c0e2bc9